Update Appendix A - Class ID Format.md (#31)
AMWA-TV/ms-05-01: docs/Appendix A - Class ID Format.md

The lineage key of a class is set of class indices that identifies the entire lineage of the class, beginning from [NcObject](https://specs.amwa.tv/ms-05-02/branches/v1.0-dev/docs/Framework.html#ncobject), extending down through all ancestor classes, and ending at the class in question. The key contains as many class indices as needed to describe the inheritance hierarchy.

To support compatible addition of non-standard classes to the standard control model, lineage keys allow _**authority keys**_ to be interposed in the sequence of class indices. This feature is described in [Non-standard classIds](#non-standard-classids).

A control class defined by the [Control framework](https://specs.amwa.tv/ms-05-02/) is called a _**standard class**_. The lineage key of a standard class contains no authority keys.

> Devices exposing vendor specific functionality would make use of non-standard classes in their device models.

Reviewer: The term "expose" here is not used in the rest of the document. Perhaps "modelling" instead? Suggested rewording:
```suggestion
Devices use non-standard classes to model vendor specific functionality in their device models.
```

• **1•2•12•7** designates class `NcXXX` , a child of the class whose parent is **1•2•12**.

## Non-standard classIds

> A class not defined by NCA is called a _**nonstandard class**_, and its classId is called a _**non-standard classId**_. A non-standard classId contains at least one authority key.

Reviewer: ```suggestion
A class not defined by NCA is called a _**non-standard class**_, and its classId is called a _**non-standard classId**_. A non-standard classId contains at least one authority key.
```
